Bottom: 2df66937ce12bf9be5e9fd41a2320cec5351d1cb
Top:    52d92b7cf4a46772e5d1095fcc0c85369fab850c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 22:06:44 +0000

Refresh of attempt-storing-arc-center

---

diff --git a/src/polygon1.c b/src/polygon1.c
index d2f63c9..802d713 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1924,7 +1924,7 @@ print_labels (PLINE * a)
 
   do
     {
-      DEBUGP ("(%mn, %mn)->(%mn, %mn) labeled %s  is_round=%i radius=%li cvc_next=%p cvc_prev=%p\n",
+      DEBUGP ("(%mn, %mn)->(%mn, %mn) labeled %s  is_round=%i radius=%f cvc_next=%p cvc_prev=%p\n",
               EDGE_BACKWARD_VERTEX (e)->point[0], EDGE_BACKWARD_VERTEX (e)->point[1],
                EDGE_FORWARD_VERTEX (e)->point[0],  EDGE_FORWARD_VERTEX (e)->point[1], theState (e),
               e->is_round, e->radius, EDGE_BACKWARD_VERTEX (e)->cvc_next, EDGE_BACKWARD_VERTEX (e)->cvc_prev);
